lib/error.js

```diff
@@ -36,7 +36,7 @@ export const makeError = ({
 	timedOut,
 	isCanceled,
 	killed,
-	parsed: {options: {timeout}},
+	parsed: {options: {timeout, cwd}},
```

Comment:
```suggestion
	parsed: {options: {timeout, cwd = process.cwd()}},
```

I am not completely sure, but I believe `parsed.options.cwd` is `undefined` there went `options.cwd` is not defined.

We should add a test case for this too.

Reply:
I kept the `undefined` and added a test for `undefined` if the user does not specify it as an option.

Would you rather go for the `process.cwd()` ? I think this is less correct, if we go that path it would be better to always set the `cwd` when running the command.

Reply:
The `cwd` option is forwarded to `child_process.spawn()` which [defaults it to `process.cwd()`](https://nodejs.org/dist/latest-v20.x/docs/api/child_process.html#child_processspawncommand-args-options).

> Use cwd to specify the working directory from which the process is spawned. If not given, the default is to inherit the current working directory.

Defaulting `error.cwd` would allow users to know the current directory that was actually used, even when the `cwd` option was not specified.